File: 수학/leetcode1523(Easy)-Count Odd Numbers in an Interval Range.py
# 나의 풀이 : 30ms(76.46%), 13.9MB(50.4%)
class Solution:
    def countOdds(self, low: int, high: int) -> int:
        n = high - low + 1

        if n % 2 == 0:
            return n // 2

        return n // 2 + 1 if low % 2 == 1 else n // 2

File: 수학/leetcode1523(Easy)-Count Odd Numbers in an Interval Range.js
// 나의 풀이 : 59ms(93.15%), 41.7MB(62.14%)
/**
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
var countOdds = function (low, high) {
  const n = high - low + 1
  if (n % 2 === 0) {
    return n / 2
  }
  return low % 2 === 0 ? Math.floor(n / 2) : Math.floor(n / 2) + 1
}
